modules/eks/aws/cert-manager.tf

```diff
@@ -15,20 +19,20 @@ module "iam_assumable_role_cert_manager" {
   number_of_role_policy_arns    = 1
   role_name                     = format("cert-manager-%s", var.cluster_name)
   provider_url                  = replace(module.cluster.cluster_oidc_issuer_url, "https://", "")
-  role_policy_arns              = [aws_iam_policy.cert_manager.0.arn]
+  role_policy_arns              = [for s in local.all_domains : aws_iam_policy.cert_manager[s].arn]
```

Comment:
Nitpicking: what does “s” stand for?

Reply:
Fixed, "s" is used in terraform documentation